src/program/2024.ts

```diff
@@ -0,0 +1,211 @@
+        description: "Er dette deg? Ta kontakt!",
+        description_en: "Is this you? Get in touch!",
+      },
+      image: "TODO: path to some image in static dir",
+      from: "2024-11-07T19:30:00",
+      to: "2024-11-07T19:50:00",
+      type: "talk",
+    },
+    {
+      title: "Lyntale",
+      title_en: "Lightning Talk",
+      description: "Ledig lyntale slot",
+      description_en: "Available lightning talk slot",
+      speaker: {
+        name: "Ukjent taler",
+        description: "Er dette deg? Ta kontakt!",
+        description_en: "Is this you? Get in touch!",
+      },
+      image: "TODO: path to some image in static dir",
+      from: "2024-11-07T19:50:00",
+      to: "2024-11-07T20:00:00",
+      type: "talk",
+    },
+    {
+      title: "Takke for kvelden",
+      description: "Takk for at du kom, de som vil delta på quiz blir igjen!",
+      image: "TODO: path to some image in static dir",
+      from: "2024-11-07T20:00:00",
+      to: "2024-11-07T20:15:00",
+      type: "info",
+    },
+    {
+      title: "Sosial mingling før quiz",
+      title_en: "Thanks",
+      description:
+        "Imens friByte og andre frivillige rydder, kan resten mingle og vente på quiz.",
+      description_en:
+        "Thank you for coming, those who wanna participate in the quiz stays behind!",
+      image: "TODO: path to some image in static dir",
+      from: "2024-11-07T20:15:00",
+      to: "2024-11-07T20:30:00",
+      type: "break",
+    },
+    {
+      title: "Pubkviss",
+      title_en: "Pubquiz",
+      description:
+        "I tradisjon tro blir det en god gammeldags quiz sesjon. Denne gang om foredragene holdt i løpet av kvelden samt IT-relaterte spørsmål!",
+      description_en:
+        "As tradition dictates, there will be a good old-fashioned quiz session. This time about the talks given during the conference as well as IT-related questions.",
+      image: "TODO: path to some image in static dir",
+      from: "2024-11-07T20:30:00",
+      to: "2024-11-07T21:30:00",
+      type: "info",
+    },
+    {
+      title: "Kvelden ferdig",
+      title_en: "The evening is finished",
+      description: "Håper du hadde en fin kveld og likte årets konferanse :)",
+      description_en:
+        "We hope you had a nice evening and liked this years iteration of the conference :)",
+      image: "TODO: path to some image in static dir",
+      from: "2024-11-07T21:30:00",
+      to: "",
+      type: "info",
+    },
+  ],
+  sponsors: [
+    {
+      name: "Kantega",
+      logo: "/sponsors/kantega.svg",
+      url: "https://www.kantega.no/",
+    },
+    {
+      name: "NAV IT",
+      logo: "/sponsors/nav.svg",
+      url: "https://www.detsombetyrnoe.no/",
+    },
+    {
+      name: "Variant",
+      logo: "/sponsors/variant.svg",
+      url: "https://www.variant.no/",
+    },
+    {
+      name: "TietoEvry",
+      logo: "/sponsors/tietoevry.svg",
+      url: "https://www.tietoevry.com/",
+    },
+  ],
+};
```

Comment:
```suggestion
} as const;
```

This will make the type even easier to parse 🤓